### ISSUE TITLE
Dumb Password Rules

### ISSUE THREAD

**Issue by djbe** [apple/password-manager-resources#91]
Saw this mentioned in a Slack conversation:
https://github.com/dumb-password-rules/dumb-password-rules

There's a whole plethora of examples that we could collect from that document, after verifying them and converting to the right format.

**Comment by igor-makarov** [comment]
This looks hilarious! 

How do you propose going about converting this list into the repo format?  
Important to consider that a lot of the rules there exist as screenshots and might have changed, so validation is important.

**Comment by djbe** [comment]
Well most contain an informal description of the rules (+ screenshots like you said), and we can't just copy them as is, we need to verify them.

I don't think we can automate this process, we'll need to do it manually. But it's at least a good reference point or list of websites with specific password rules that fit this project.

**Comment by igor-makarov** [comment]
I agree, this resource seems very useful, even if some of these rules can't even be represented with the current grammar.

I think the first step might be to go over the list and conduct a preliminary triage.  
My suggestion for important questions to ask about each website:
1. How critical is to include the website in terms of popularity, risk of using insecure passwords, etc? 
2. How well can the current grammar cover the website's password quirks? Can it cover all of them? Which of the rules do not need coverage because a password manager would comply with them by default?
3. Is the website already covered in our repo? Does it actually need coverage? I see `waze.com` on the list and it seems like the problem with it is that the password rules aren't visible before entry, not that they are bad per se. Also, `citi.com` is already added for example.

**Comment by duffn** [comment]
Hi, I made dumb password rules and would be very happy if we could get the rules converted.

> Important to consider that a lot of the rules there exist as screenshots and might have changed, so validation is important.

Yes, indeed! The rules have not been checked after initial submission, so need validation. 

> I think the first step might be to go over the list and conduct a preliminary triage.

I think that is a fine plan.

**Comment by igor-makarov** [comment]
@duffn - how about you and @djbe go over the list together and make some decisions about what needs to be added and with what priority?

**Comment by rmondello** [comment]
(I just want to quickly thank everyone in this thread for being excited about this.)

**Comment by hassankhan** [comment]
Hi all, I've started converting https://github.com/dumb-password-rules/dumb-password-rules/ to the Password Rules format.

Unfortunately I won't be able to verify all of them, should that be a precondition for the PR? Also, should I create individual PRs for each site?

**Comment by rmondello** [comment]
Can we start with things you can verify in one PR first, and then move onto individual PRs for things you cannot?

**Comment by hassankhan** [comment]
@rmondello Absolutely 👍 just so you know I don't have accounts with the vast majority of them 😅
